tests/integration/test_schema.py

```diff
@@ -1177,11 +1185,135 @@ async def test_schema(registry_async_client, trail):
     assert res.json()["error_code"] == 40402
     assert res.json()["message"] == "Version 15 not found."
 
+
+async def test_schema_subject_post_invalid(registry_async_client):
+    """
+    Tests posting to /subjects/{subject} with different invalid values.
+    """
+    subject_name_factory = create_subject_name_factory("test_schema_subject_post_invalid")
+
+    schema_str = '{"type": "string"}'
+
+    # Create the subject
+    subject_1 = subject_name_factory()
+    res = await registry_async_client.post(
+        "subjects/{}/versions".format(subject_1),
+        json={"schema": schema_str},
+    )
+    assert res.status == 200
+
+    # Invalid schema for existing subject should return 500
```

Comment:
please use this as the `assert` message below. it makes it easier to read test failures

Reply:
Fixing!